### PR TITLE
moving all developer-focused docs into EMOD code repo

### PR DESCRIPTION
**Pull request by JSchripsema-IDM**
This is currently building without error, but only contains links back to the generic EMOD docs. Links to the HIV and malaria docs are in comments until they can be merged into emodpy-hiv and emodpy-malaria. Once those are merged, I can update those commented links and this PR can be reviewed.

### CODE REVIEW

**Review by Bridenbecker — CHANGES_REQUESTED** [review]
There are just a couple of small things to clean up.  Thank you for saving these docs.  I did not know that some of it was even there.

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,452 @@
...
+
+
+# Example configuration for intersphinx: refer to the Python standard library.
+intersphinx_mapping = {'python': ('https://docs.python.org/3', None),
+                       'emod-api': ('https://docs.idmod.org/projects/emod-api/en/latest/', None),
+                       'emodpy': ('https://docs.idmod.org/projects/emodpy/en/latest/', None),
+                       'emodpy_generic': ('https://docs.idmod.org/projects/emodpy-generic/en/latest/', None),
+                       'idmtools': ('https://docs.idmod.org/projects/idmtools/en/latest/', None),
+                       'calibra': ('https://docs.idmod.org/projects/idmtools_calibra/en/latest/', None),
+                       'emod-hiv': ('https://docs.idmod.org/projects/emod-hiv/en/latest/', None),
```
[the remark this answers is not in the document]
Comment (JSchripsema-IDM) [reply docs/conf.py]: I just missed it, removed now.

File: docs/_templates/footer_start.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<p>emodpy-hiv is licensed under the <a href="https://creativecommons.org/licenses/by-sa/4.0/legalcode">Creative Commons
+    Attribution-ShareAlike 4.0 International License</a>.</p>
+<p>Send documentation feedback to <a href="mailto:idmsupport@gatesfoundation.org" target="_top">idmsupport@gatesfoundation.org</a>.</p>
```
[the remark this answers is not in the document]
Comment (JSchripsema-IDM) [reply docs/_templates/footer_start.html:2]: Fixed this, docset name, and license.

File: README.md [hunk 3]
```diff
@@ -85,7 +86,7 @@ https://docs.idmod.org/projects/emod-malaria/en/latest/dev-install-overview.html
 
 Contributing
 ------------
-Questions or comments can be directed to [support@idmod.org](<mailto:support@idmod.org>).
+Questions or comments can be directed to [support@idmod.org](<mailto:idm@gatesfoundation.org>).
```
[the remark this answers is not in the document]
Comment (JSchripsema-IDM) [reply README.md]: Just a mistake on my part!

**Comment by JSchripsema-IDM** [comment]
@Bridenbecker Thanks Dan for the thorough review! I addressed all of your comments. Once all three related EMOD PRs are merged, I'll update all the links that were commented out.

**Comment by Bridenbecker** [comment]
@JSchripsema-IDM  FYI.  I'm assuming that **you** will merge when you think it is ready.

**Pull request merged**